base_utils.py

```diff
@@ -263,5 +263,6 @@ async def process_with_overlap(req, attribute_service, POST_PROMPT, Posts):
             )
             all_dict_meta+= result['post_meta']
```

Comment:
Dòng này đang in ra chế độ debug, nên đổi thành production:
```suggestion
print("Production mode tuantuan")
```